pydra/engine/tests/test_node_task.py

```diff
@@ -769,12 +781,11 @@ def test_task_state_1(plugin_dask_opt, input_type, tmpdir):
         assert odir.exists()
 
 
-def test_task_state_1a(plugin, tmpdir):
+def test_task_state_1a(plugin, tmp_path):
     """task with the simplest splitter (inputs set separately)"""
     nn = fun_addtwo(name="NA")
-    nn.split(splitter="a")
-    nn.inputs.a = [3, 5]
```

Comment:
I understand that setting input later is a still valid option, right?

Reply:
actually, I guess it is not...

Reply:
> actually, I guess it is not...

You can but you need to wrap the input in a `StateArray` object, e.g.

`nn.inputs.a = specs.StateArray([3, 5])`

Reply:
so, perhaps we can use it in the test, otherwise `test_task_state_1a` is the same as `test_task_state_1`